apps/designer/app/designer/features/style-panel/controls/toggle/toggle-control.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  styled,
+  Flex,
+  Tooltip,
+  ToggleGroupRoot,
+  ToggleGroupItem,
+} from "@webstudio-is/design-system";
+import { StyleProperty } from "@webstudio-is/react-sdk";
+import { useIsFromCurrentBreakpoint } from "../../shared/use-is-from-current-breakpoint";
+
+export type ToggleGroupControlProps = {
+  property: StyleProperty | StyleProperty[];
+  value: string;
+  items: { child: JSX.Element; label: string; value: string }[];
+  onValueChange?: (value: string) => void;
+};
+
+export const ToggleGroupControl = ({
+  property,
+  value = "",
+  items = [],
+  onValueChange,
+}: ToggleGroupControlProps) => {
+  const isCurrentBreakpoint = useIsFromCurrentBreakpoint(property);
+  return (
+    <ToggleGroupRoot
+      type="single"
+      value={value}
+      onValueChange={onValueChange}
+      css={{ width: "fit-content" }}
+    >
+      {items.map(({ child, label, value }, index) => {
+        return (
+          <ToggleGroupControlItem
+            key={index}
+            value={value}
+            state={isCurrentBreakpoint ? "set" : undefined}
```

Comment:
"set" is also application level thing, sorry, you either need to add that state to a new styled component like a last time, or it's something that we need everywhere and its not "on" state it already has, then you have to explain what use case it has somewhere else

Reply:
ohhh its local component now, ignore  that comment above